.github/workflows/python.yaml

```diff
@@ -0,0 +1,31 @@
+name: Python package
+
+on: [push]
```

Comment:
Optional but if you want the ability to click a button to run the workflow on demand (e.g. for testing) you can change this to:

```yaml
on:
  push:
  workflow_dispatch:
```

Reply:
Cool!  I have so much to learn about GH Actions.